### ISSUE TITLE
Small Screen Layout

### ISSUE THREAD

**Issue by wormuths** [rejuvenate/lovelace-horizon-card#80]
Hi,

I was just wondering if there was any change that could be done to better layout the card on a small screen such as a cell phone?

I have a two-column section that your card is in, and on cell phones, it doesn't rearrange to fill the section like other cards (weather) next to it does.  Maybe we could have the "Solar Noon" section appear on its own line in that scenario, so the three values (dawn, solar noon, dusk) appear over two rows which would extent it lower to fill the blank space, and declutter the text a little bit.  Not sure what that would involve...

Here's an example of how it appears, and what I'd like to see improved, if possible.

![Layout Bug](https://github.com/rejuvenate/lovelace-horizon-card/assets/47436307/0d56003f-e5c8-436b-af42-8b7b916dff75)

Thanks for all the work...

**Comment by Nickduino** [comment]
Agreed. It's a beautiful card but its (useful information)/(real estate taken) could be improved. **Making it compatible with section and the grid layout would help**.
